README.markdown

```diff
@@ -527,6 +527,28 @@ func reticulateSplines(spline: [Double], adjustmentFactor: Double,
 }
 ```
 
+Since [SE-0091](https://github.com/apple/swift-evolution/blob/master/proposals/0091-improving-operators-in-protocols.md), where applicable, declare an operator function as a static function within the type it is relevant to.
```

Comment:
The discussion in this link is rather interesting. To me, especially the Subclass problem 
```class Subclass: Superclass {
  var bar: String

  static func ==(lhs: Subclass, rhs: Subclass) -> Bool {
    guard lhs as Superclass == rhs as Superclass else {
      return false
    }
    return lhs.bar == rhs.bar
  }
}
```